bin/hyrax-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -e
+
+# Make sure common volume points have the app permission
+chown -fR app:app /app/samvera/hyrax-webapp/tmp
+chown -fR app:app /app/samvera/hyrax-webapp/public
+
+mkdir -p /app/samvera/hyrax-webapp/tmp/pids
+rm -f /app/samvera/hyrax-webapp/tmp/pids/*
+
+if [ -z $RAILS_ENV ]
```

Comment:
this can cause surprising results in specs if you left on.  I recommend taking it out until / unless we need it set for something specific.

Reply:
fixed in c186c99